Fix The Usage Section in Docs (#837)
uclahs-cds/package-moPepGen: docs/filter-fasta.md

### Filter by Expression and Miscleavages

> This example is the same as above with the addition of a filter by miscleavages. Any peptides with miscleavages larger than 3 will be dropped.

Reviewer: ```suggestion
This example is the same as above with the addition of a filter by miscleavages. Any peptides with more than 3 miscleavages will be dropped.
```

### Filter Variant and Noncoding Peptides

> This example takes both a variant peptide and noncoding peptide FASTA file and filters the sequences based on the expression level of the transcripts that they are associated with.

Reviewer: ```suggestion
This example takes both the variant peptide FASTA and the noncoding peptide FASTA and filters the peptides based on the expression level of the transcripts they are associated with.
```

> This example here removes any peptide sequences if they appear in the given denylist.

Reviewer: ```suggestion
This example here removes any peptide sequences that appear in the given denylist.
```

!!! warning:

> When using noncoding peptides in a denylist, do not also pass it as an input FASTA, because all peptides will be removed.

Reviewer: ```suggestion
When using noncoding peptides in a denylist, do not also pass the noncoding peptide FASTA as an input FASTA, because all peptides will be removed.
```

### Complex Filtering

> Sometimes we want a more complex filtering strategy. In the example below, we want to first remove any variant peptides that overlap with any noncoding peptide, and next we filter again based on the expression level.

Reviewer: ```suggestion
Sometimes we want a more complex filtering strategy. In the example below, we want to first remove any variant peptides that overlap with any noncoding peptides, and then filter again based on the expression level.
```
